bin/i18n/src/i18n/create_artifacts/backend.clj

```diff
@@ -0,0 +1,48 @@
+(ns i18n.create-artifacts.backend
+  (:require [clojure.java.io :as io]
+            [clojure.string :as str]
+            [i18n.common :as i18n]
+            [metabuild-common.core :as u])
+  (:import [java.io FileOutputStream OutputStreamWriter]
+           java.nio.charset.StandardCharsets))
+
+(defn- backend-message? [{:keys [source-references]}]
+  (some (fn [path]
+          (some
+           (fn [dir]
+             (str/starts-with? path dir))
+           ["src" "backend" "enterprise/backend"]))
```

Comment:
do we need shared here?

Reply:
oh, in shared does that need to be in backend and frontend?

Reply:
Good point. Shared should probably go here. Altho I don't know how we're going to i18n things in shared right now, since `metabase.util.i18n` isn't going to work in ClojureScript. I guess we can make a cljs version that compiles to whatever we use on the FE (`ttag`?)